build/bin/build-server-docker

```diff
@@ -10,24 +10,18 @@ if [ -f "${docker_tarball}" ]; then
     exit 0
 fi
 
+cp -r ${root_dir}/sources/percona-images/ansible ${rpms_dir}/../ansible
+
+# enable 'local' repo in ansible playbooks
+sed -i -e "s/enablerepo: \"pmm2-server\"/enablerepo: local/" ${rpms_dir}/../ansible/roles/pmm2-images/tasks/main.yml
+sed -i -e "s/pmm_client_repo_name: \"percona-testing-x86_64\"/pmm_client_repo_name: local/" ${rpms_dir}/../ansible/roles/pmm2-images/vars/main.yml
```

Comment:
I think we should leave comments near things that are `sed`'ed because it makes them very fragile during refactoring.